### PR TITLE
PLANET-3380 admin convert blocks

### PR DESCRIPTION
**Pull request by angtheod**
Add Menu Item for Blocks and a submenu item. The Blocks Usage logic was moved to the submenu item and in the main menu item we added a button and an input for Editors to use for replacing block shortcodes with new ones.

*`class-blocks-usage-controller.php` has the code that was in plugin-blocks.php (related to block usage report functionality)... so no real changes on this one, only restructured so that it becomes a submenu item.

### CODE REVIEW

**Comment by angtheod** [comment]
Works with shell_exec, but I am trying to make it work with WP_CLI::runcommand for more safety.
image

File: classes/controller/menu/class-settings-controller.php [hunk 1]
```diff
@@ -32,47 +34,84 @@ public function create_admin_menu() {
...
+
+			$data['nonce_action'] = 'convert-blocks-nonce';
+			$data['form_submit']  = 0;
+
+			if ( 'POST' === $_SERVER['REQUEST_METHOD'] ) {
+				$data['form_submit'] = 1;
+
+				if ( ! wp_verify_nonce( $nonce, 'convert-blocks-nonce' ) ) {
+					$data['message'] = __( 'Nonce verification failed!', 'planet4-blocks-backend' );
+					return false;
```
Comment (sagarsdeshmukh) [inline classes/controller/menu/class-settings-controller.php]: If nonce verification fails, the error message will not display because of return false.

**Comment by sagarsdeshmukh** [comment]
@angtheod - On my local env. the Rest API end point(https://www.planet4.test/wp-json/plugin_blocks/v1/plugin_blocks_report/) throughs 404. could you please confirm it once.

**Comment by comzeradd** [comment]
Please make sure to squash the last two commits (the fixes). Either locally or here on merge.

**Pull request merged**